.github/workflows/perf.yml

```diff
@@ -30,52 +30,78 @@ jobs:
           mkdir -p build
           cd build && cmake -DUSE_BUNDLED_DEPS=False ../
           make unit-test-libsinsp -j4
+          make sinsp-example -j4
 
-      - name: Run Perf
+      - name: Run Perf - unit tests
         shell: bash
         run: |
           cd build
-          sudo perf record --call-graph dwarf -o perf.data -q libsinsp/test/unit-test-libsinsp
+          sudo perf record --call-graph dwarf -o perf_tests.data -q libsinsp/test/unit-test-libsinsp
+
+      - name: Run Perf - scap file
+        shell: bash
+        run: |
+          cd build
+          wget https://download.falco.org/fixtures/trace-files/traces-positive.zip
+          unzip traces-positive.zip
+          sudo perf record --call-graph dwarf -o perf_scap.data -q ./libsinsp/examples/sinsp-example -s traces-positive/falco-event-generator.scap   
 
       - name: Archive master perf report
         if: github.event_name == 'push'
         uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
         with:
           name: perf_report
           retention-days: 30 # 30 days because this is the artifact on master; we need to retain it to be able to properly diff it
-          path: build/perf.data
+          path: build/perf_*.data
           if-no-files-found: error
 
       - name: Download latest master report
         if: github.event_name == 'pull_request'
         uses: dawidd6/action-download-artifact@v6
         with:
-          workflow: ci.yml
+          workflow: perf.yml
           branch: master
           event: push
           name: perf_report
 
-      - name: Diff from master
+      - name: Diff from master - unit tests
         if: github.event_name == 'pull_request'
         run: |
-          sudo perf diff perf.data build/perf.data -d unit-test-libsinsp -b -o 0 --percentage relative -q &> perf_diff.txt
+          sudo perf diff perf_tests.data build/perf_tests.data -d unit-test-libsinsp -b -o 1 --percentage relative -q &> perf_tests_diff.txt
```

Comment:
Switched to `-o 1` ordering that means to order the diff by absolute difference, descendent